fuzzing/oss-fuzz-scripts/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+python3 -m pip install .
+
+# Directory to look in for dictionaries, options files, and seed corpa:
+SEED_DATA_DIR="$SRC/seed_data"
```

Comment:
> I _think_ `SRC` comes from outside this script and is set in such a way that it is very unlikely to start with `-`.

You are correct. I explained the OSS-Fuzz provided env vars a bit more in this comment: https://github.com/gitpython-developers/GitPython/pull/1901#issuecomment-2063818998. These variables are certainly a case where `--` is _truly_ known not to be needed, because if the omission of it results in these scripts breaking then most (more likely, all) of the other OSS-Fuzz projects would break as well.

---

That said, I wasn't particularly happy with having the `"$SRC/seed_data"` directory specified in two separate scripts, where one implicitly depends on the other creating it. If nothing else, that feels typo prone.

Initially I considered having a third script that just defined the shared variable and `source`ing it in each, but that felt like overkill.

Ultimately, I think the best thing to do would be to remove the need for that directory all together. When I get around to the changes I described in a reply to you here: https://github.com/gitpython-developers/GitPython/pull/1903#issuecomment-2062628214, I should be able to do all of the dictionary & corpus zip creation inside `container-environment-bootstrap.sh` (i.e., during the Docker image build step) and put everything in `$OUT/` directly from there. That would allow the removal of the `"$SRC/seed_data"` directory completely, and also make the fuzz harness build step more efficient because it wouldn't need to produce the fixture artifacts every time.

I'm open to alternative suggestions though 🙂 

I'll leave this comment unresolved for now for my own reference